src/app/api/audit-log/calls/route.ts

```diff
@@ -0,0 +1,53 @@
+
+import {NextResponse, NextRequest} from 'next/server';
+
+
+interface FetchedCalls {
+    id: string;
+    direction: string;
+    from: string | null;
+    to: string | null;
+    timestamp: string;
+}
+
+export async function GET(
+    request: NextRequest,
+    res: NextResponse
+) {
+   const accountSid = process.env.TWILIO_ACCOUNT_SID;
+   const authToken = process.env.TWILIO_AUTH_TOKEN;
+   const client = require('twilio')(accountSid, authToken);
+   try{
+    console.log('success')
+    const calls: any[] = await client.calls.list({limit:20});
+    //if want to display all calls, can delete the limit parameter
+    const formattedCalls: FetchedCalls[] = calls.map(call => {
+        const date = new Date(call.dateCreated);
```

Comment:
would you be able to move the date formatting to a util function and use the dayjs package for formatting?